code/client/launcher/Main.cpp

```diff
@@ -386,10 +398,18 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, PWSTR pCmdLine
 
 	if (!toolMode)
 	{
-		if (OpenMutex(SYNCHRONIZE, FALSE, L"CitizenFX_LogMutex") == nullptr)
+		const auto handle_deleter = [](HANDLE handle){ if (handle != INVALID_HANDLE_VALUE) CloseHandle(handle); };
+		using UniqueHandle = std::unique_ptr<std::remove_pointer<HANDLE>::type, decltype(handle_deleter)>;
+
+		UniqueHandle mutexHandle(
+			OpenMutex(SYNCHRONIZE, FALSE, L"CitizenFX_LogMutex_Mod"), 

```

Comment:
_Mod? Also, I'm not sure if it makes sense to close this handle upon scope ending - this logic is entirely unused I believe rather nowadays (we don't have .log.X files to rotate anymore) but if this were still used subprocesses would rotate logs a lot.